### ISSUE TITLE
Incorrect `norm(::RodriguesParam)`

### ISSUE THREAD

**Issue by hyrodium** [JuliaGeometry/Rotations.jl#187]
```julia
julia> using Rotations, LinearAlgebra

julia> R = RodriguesParam(1., 2., 3.)
3×3 RodriguesParam{Float64} with indices SOneTo(3)×SOneTo(3)(1.0, 2.0, 3.0):
 -0.733333  -0.133333  0.666667
  0.666667  -0.333333  0.666667
  0.133333   0.933333  0.333333

julia> norm(R)
3.7416573867739413

julia> norm(collect(R))
1.732050807568877
```
The last two returned values should be equal.

**Comment by hyrodium** [comment]
And also `norm(::MRP)`.
```julia
julia> R = MRP(1., 2., 3.)
3×3 MRP{Float64} with indices SOneTo(3)×SOneTo(3)(1.0, 2.0, 3.0):
  0.537778   0.764444   -0.355556
 -0.622222   0.644444    0.444444
  0.568889  -0.0177778   0.822222

julia> norm(R)
3.7416573867739413

julia> norm(collect(R))
1.7320508075688772
```

**Comment by hyrodium** [comment]
```julia
julia> R = UnitQuaternion(1., 2., 3., 4.)
3×3 UnitQuaternion{Float64} with indices SOneTo(3)×SOneTo(3)(0.182574, 0.365148, 0.547723, 0.730297):
 -0.666667   0.133333  0.733333
  0.666667  -0.333333  0.666667
  0.333333   0.933333  0.133333

julia> norm(R)
0.9999999999999999

julia> norm(collect(R))
1.732050807568877
```

**Comment by hyrodium** [comment]
Ah, these will be fixed in #175.
https://github.com/JuliaGeometry/Rotations.jl/pull/175/files#diff-03ef4407ea8e9dce235469690db4a92112040c3ebb12c89b33543eeea631601eL51

**Comment by hyrodium** [comment]
This issue has been solved in #175.
